superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -383,11 +422,11 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
             <div className="helper">
               {t('Refer to the ')}
               <a
-                href={conf?.SQLALCHEMY_DOCS_URL ?? ''}
```

Comment:
remove these diffs, we should be referencing `conf` for `SQLALCHEMY_DOCS_URL` and `SQLALCHEMY_DISPLAY_TEXT`